scripts/barclamp_mgmt_lib.rb

```diff
@@ -260,6 +260,8 @@ def generate_assets_manifest
 
   manifests = Pathname.new(CROWBAR_PATH).join("barclamps", "manifests")
 
+  manifests.mkdir unless manifests.exist?
```

Comment:
> Oh sorry I mean read. https://github.com/rsalevsky/crowbar/blob/create_dir/scripts/barclamp_mgmt_lib.rb#L267 fails

So we're back to my first comment, why not simply return instead of doing nothing in the method?